src/Components/Header/Header.js

```diff
@@ -43,14 +49,19 @@ const Header = () => {
     return dropdownMenuItems;
   };
 
+  const referrer = formData.immutableReferrer ? formData.immutableReferrer : 'default';
+  const logoInfoArray = getReferrerInfo(Number(config?.referrerData?.logoSource));
+  const logoSource = logoInfoArray ? logoInfoArray[0][referrer] : defaultLogo;
+  const logoAlt = logoInfoArray ? logoInfoArray[1][referrer]: coLogoAlt.default;
```

Comment:
Make this so that it uses `getReferrer`.

For example:
```js
const logoSource = getReferrer('logoSource');
const logoAlt = getReferrer('logoAlt');
```